### PR TITLE
Barry merging

### PR DESCRIPTION
**Pull request by bmckernan**

### CODE REVIEW

**Comment by bmckernan** [comment]
I'm merging this, but should request SF to ok this in future.

**Review by bmckernan — COMMENTED** [review]
These are changes to include retro switch to convert retro BBH to pro BBH. Plus, can handle multiple binaries, but need to complete treatment of binaries as array of binaries throughout. Plus clean array of repeat singletons in multiple binaries.

**Comment by saavikford** [comment]
This is fine--B added important switches and fixed several bugs in terms of binary handling.

**Comment by saavikford** [comment]
I think I didn't approve the merge when I wanted to???

**Review by saavikford — APPROVED** [review]
Good--fixed bugs, added retro on/off switch.

File: inputs/model_choice.txt
Comment (saavikford) [inline inputs/model_choice.txt:1]: This is adding the choice of retro binaries on/off in the input file (good)

**Review by saavikford — APPROVED** [review]
This is fine... but it won't merge? Maybe it will now??

**Pull request merged**